python/tvm/relax/frontend/onnx_frontend.py

```diff
@@ -552,7 +652,19 @@ def _convert_operator(self, op_name, inputs, attrs, opset):
         """
         convert_map = _get_convert_map(opset)
         if op_name in convert_map:
-            sym = convert_map[op_name](self.bb, inputs, attrs)
+            convert_class = convert_map[op_name]
+            op_function = convert_class.get_converter(opset)
+            # If the op_function is a subclass of RelayOnnxOpConverter then it is a relay op.
+            if (convert_class.__bases__[0] == RelayOnnxOpConverter) or \
+                (hasattr(convert_class.__bases__[0], "__bases__") and convert_class.__bases__[0].__bases__[0] == RelayOnnxOpConverter):
```

Comment:
this second condition is very opaque - what's it handling?

Reply:
it was checking for second degree subclassing. I replaced this with `issubclass(convert_class, RelayOnnxOpConverter)`